dataactbroker/handlers/accountHandler.py

```diff
@@ -387,6 +387,10 @@ def updateUser(self, system_email):
         if requestDict.exists("is_active"):
             is_active = bool(requestDict.getValue("is_active"))
             if not self.isUserActive(user) and is_active:
+                # Reset password count to 0
+                self.resetPasswordCount(user)
+                # Reset last login date so the account isn't expired
+                self.interfaces.userDb.updateLastLogin(user, unlock_user=True)
```

Comment:
Is there anything currently present we can use to check if they've been unlocked?  Maybe add an unlock_date that we can check as an alternative before locking due to inactivity?